src/katgpucbf/xbgpu/engine.py

```diff
@@ -420,6 +420,13 @@ def __init__(
 
         self._populate_sensors()
 
+    async def _get_in_item(self) -> InQueueItem | None:
+        """Get the next :class:`InQueueItem`.
+
+        This is wrapped in a method so it can be mocked.
+        """
+        return await self._in_queue.get()
+
```

Comment:
Argh, apologies. I thought I put it back where I found it. Will move it once more.